nixos/modules/services/x11/window-managers/xmonad.nix

```diff
@@ -37,7 +37,7 @@ let
 
   xmonad = if (cfg.config != null) then xmonad-config else xmonad-vanilla;
 in {
-  meta.maintainers = with maintainers; [ lassulus xaverdh ivanbrennan ];
+  meta.maintainers = with maintainers; [ lassulus xaverdh ivanbrennan slotThe ];
```

Comment:
Split this in a separate commit (no need for a new PR).

Reply:
Done, thanks!